.github/workflows/send-failure-summary-notification.yml

```diff
@@ -0,0 +1,34 @@
+name: Send failure summary notification
+
+on:
+  workflow_dispatch:
+    inputs:
+      job_failures:
```

Comment:
Would be nice to have an example showing what the input looks like. That's always useful to understand manual workflow